src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -365,15 +365,17 @@ def _create_kubernetes(resource_group_name, deployment_name, dns_name_prefix, na
     smc = _resource_client_factory()
     return smc.deployments.create_or_update(resource_group_name, deployment_name, properties)
 
-def acs_get_credentials(dns_prefix, location):
+def acs_get_credentials(dns_prefix, location, output_file=None):
     # TODO: once we get the right swagger in here, update this to actually pull location and dns_prefix
     #acs_info = _get_acs_info(name, resource_group_name)
-    home = os.path.expanduser('~')
 
-    path = os.path.join(home, '.kube', 'config')
+    if not output_file:
+        home = os.path.expanduser('~')
+        output_file = os.path.join(home, '.kube', 'config')
```

Comment:
This logic for setting the default value for the parameter should go in the _params.py file.

Main advantages are:
1. When a user does --help on the command, they'll see what the default value will be.
2. You can also add tab completion so they can specify their own path.

So in _params.py, something like:
```
register_cli_argument('acs kubernetes get-credentials', 'output_file', help='<APPROPRIATE HELP TEXT HERE', default=os.path.join(os.path.expanduser('~'), '.kube', 'config'), completer=FilesCompleter())
```
This is similar to https://github.com/Azure/azure-cli/blob/master/src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_params.py#L60